frameworks/V/pico.v/pico.v-optimized.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM syou/v-dev:0.1
+
+WORKDIR /app
+
+COPY ./main.v run.sh ./
+
+RUN git clone https://github.com/S-YOU/pico.v src && cd src && git checkout v0.0.4 \
+    && cd /app/src/picoev && git clone https://github.com/S-YOU/picoev src && cd src && git checkout v0.0.1 \
+    && cd /app/src/picohttpparser && git clone https://github.com/S-YOU/picohttpparser src && cd src && git checkout v0.0.1 \
```

Comment:
Please don't clone all the git repo, for later checkout a branch. It's very slow.

Clone only the branch:
`git clone -b v0.0.1 --single-branch --depth 1 https://github.com/S-YOU/picoev`